TeamMentor/src/main/java/org/firstinspires/ftc/teamMentor/ConceptTensorFlowObjectDetection.java

```diff
@@ -84,9 +84,7 @@ public class ConceptTensorFlowObjectDetection extends LinearOpMode {
      * and paste it in to your code on the next line, between the double quotes.
      */
     private static final String VUFORIA_KEY =
-            //" -- YOUR NEW VUFORIA KEY GOES HERE  --- ";
-    // Vuforia licence key - Truong
-            "AdTQ1zz/////AAABmcsU0JSsfUAAnZVqALQznloFPRzK4IDs9AKHiU80F9ncKlBHZBPTN3XWUSLbcsKUfy/iW4P/y64OCRHrAiUTE430LhFnx8rGRtKUv8P03XTaE11Xj9gbN5vThAIBcrnk/CovUIBFJjptCseciz/akh2mWHAlNznx5kWdP0QbFRi9i6fZffoHXaBXYERvzyK/wYYxMLuwVL+qBGIuRzJRS4f2b6RZ8cq/SEs6Ulfg5HQgV24KqFA65+T7iGXKCrdQMi0eUN0Oc4DmrKrHKF55bEtA108/jh8cz1tAwsrAvjle6JUX+yUQ4RDX8Zv/GpuWdek3VFGxumvh8EdQAmZqUmaWdcrpHLXMcftKdOjyvyUf";
+            " -- YOUR NEW VUFORIA KEY GOES HERE  --- ";
```

Comment:
Please don't delete the vuforia key in the mentor folder. Set this file back to how it was.

Reply:
Done!